File: js/pigLatin.js
function translate(word) {
  let output = word;
  vowels = ['a','e','i','o','u'];
  let in_vowels = false;
  for (let j = 0; j < vowels.length; ++j) {
    if (word[0] == vowels[j]) {
      in_vowels = true;
    }
  }
  if (in_vowels == true) {
    output += 'ay';
  }
  else {
    let move = word[0];
    output = '';
    for (let i = 1; i < word.length; ++i) {
      output += word[i]
    }
    output += move;
    output += 'ay';
  }
  return output;
};

module.exports = translate;

File: python/pig_latin.py
def translate(word_or_phrase):
  output = word_or_phrase
  vowels = ['a','e','i','o','u']
  if word_or_phrase[0] in vowels:
    output += 'ay'
  else:
    move = word_or_phrase[0]
    output = word_or_phrase[1:]
    output += move
    output += 'ay'

  return output





